actionmailbox/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+*   Add ability to configure ActiveStorage service
+    for storing email raw source
+
```

Comment:
```suggestion
    for storing email raw source.

```

(Minor sentence period to follow previous conventions.)

Reply:
Done